banzai/qc/header_checker.py

```diff
@@ -0,0 +1,155 @@
+"""
+This module performs controls on several critical parameters of image header, contains in
```

Comment:
I would rephrase this to be something like "This module performs basic sanity checks that the main image header keywords are the correct format and validates their values."